app/src/test/java/http_server/ResponseFormatterTest.java

```diff
@@ -29,11 +27,12 @@ public void testBuildResponse() {
 
     String actualResponse = responseFormatter.format(response);
 
-    assertThat(actualResponse, startsWith("HTTP/1.1 200 OK\r\n"));
-    assertThat(actualResponse, containsString("Content-Type: text/plain\r\n"));
-    assertThat(actualResponse, containsString("Server: Test Server\r\n"));
-    assertThat(actualResponse, containsString("\r\n"));
-    assertThat(actualResponse, containsString("Hello, world!"));
+    assertThat(actualResponse).startsWith("HTTP/1.1 200 OK\r\n");
+    assertThat(actualResponse).contains("Content-Type: text/plain\r\n");
+    assertThat(actualResponse).contains("Server: Test Server\r\n");
+    assertThat(actualResponse).contains("\r\n");
+    assertThat(actualResponse).contains("Hello, world!");
```

Comment:
Not a requirement but you can actually chain your assertions, which is really neat about AssertJ! For example:
```
assertThat(actualResponse)
   .startsWith("HTTP/1.1 200 OK\r\n");
   .contains("Content-Type: text/plain\r\n");
   .contains("Server: Test Server\r\n");
   .contains("\r\n");
   .contains("Hello, world!");
```